src/NzbDrone.Core/SkyhookNotifications/SkyhookNotificationProxy.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using NLog;
+using NzbDrone.Common.Cloud;
+using NzbDrone.Common.EnvironmentInfo;
+using NzbDrone.Common.Http;
+
+namespace NzbDrone.Core.SkyhookNotifications
+{
+    public interface ISkyhookNotificationProxy
+    {
+        List<SkyhookNotification> GetNotifications();
+    }
+
+    public class SkyhookNotificationProxy : ISkyhookNotificationProxy
```

Comment:
I didn't want it as ServicesNotifications as it's more easily confused and simply picked Skyhook as working title I'm open for suggestions. Notifications is already taken, Blog isn't it, Feed nope, Updates, already taken....